src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -633,6 +716,9 @@ double Mis_GetTimeStep_CCSN( const int lv, const double dTime_dt )
       dt_CCSN = fmin( dt_CCSN, dt_LB );
    }
 
+   if ( !CCSN_Is_PostBounce  &&  SrcTerms.Deleptonization )
+      dt_CCSN = MIN(  dt_CCSN, Mis_GetTimeStep_CoreCollapse( lv, dTime_dt )  );
```

Comment:
Use `fmin` instead since `dt_CCSN` is double.

Reply:
I changed it to `fmin`